teacher.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person_spec'
+
+class Teacher < Person
+  def initialize(name, age, parent_permission, specialization)
+    super(name, age, parent_permission)
+    @specialization = specialization
+  end
```

Comment:
- [ ] As mentioned previously kindly make sure to add a default value for the `name` and `parent_permission` parameters for the Teacher class as well.